pdc/apps/compose/lib.py

```diff
@@ -197,7 +197,7 @@ def compose__import_rpms(request, release_id, composeinfo, rpm_manifest):
                     imported_rpms += 1
                     path, filename = os.path.split(rpm_data['path'])
                     rpm_id = get_or_insert_rpm(rpms_in_db, cursor, rpm_nevra, srpm_nevra, filename)
-                    sigkey_id = common_models.SigKey.get_cached_id(rpm_data["sigkey"], create=True)
+                    sigkey_id = common_models.SigKey.get_cached_id(rpm_data["sigkey"])
```

Comment:
Do you know what happens if you try to import compose with new sigkey and it does not exist yet? Is the error message good enough to explain what went wrong?

Reply:
I didn' t try the compose, but I try the release endpoint, it will raise error like below.
{"sigkey":["Object with key_id=abcd12371111 does not exist."]}
So I think this error is enough to understand, and so the same as "compose" endpoint.
What's your thinking?